aws/data_source_aws_workspaces_image_test.go

```diff
@@ -0,0 +1,40 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataSourceAwsWorkspacesImage_basic(t *testing.T) {
+	dataSourceName := "data.aws_workspaces_image.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAwsWorkspacesImageConfig("wsi-vwtykwqyx"),
```

Comment:
This won't pass in our testing environments, since the image is owned by your account.

We could add an environment variable for the image ID and have a pre-check function that checks for the environment variable. Alternatively, there are API functions to import AMIs as Workspace images.

Reply:
I like the idea with environment variable, will add the pre-check for now. Unfortunately, [ImportWorkspaceImage API](https://docs.aws.amazon.com/workspaces/latest/api/API_ImportWorkspaceImage.html) works only with Windows. I've submitted respective feature request to AWS Support last year. Is it OK to create `aws_workspace_image` Terraform resource only for Windows, while AWS supports Linux images too?

Reply:
Yes, we can do that. We'll just need to document that it only supports Windows images because of the AWS API

Reply:
Unfortunately, even Windows images could be imported only with [BYOL](https://docs.aws.amazon.com/workspaces/latest/adminguide/byol-windows-images.html), which looks like a totally separate feature.